server/util.go

```diff
@@ -348,6 +349,8 @@ func dumpTextValue(mysqlType uint8, value types.Datum) ([]byte, error) {
 		return hack.Slice(value.GetMysqlEnum().String()), nil
 	case types.KindMysqlSet:
 		return hack.Slice(value.GetMysqlSet().String()), nil
+	case types.KindMysqlJSON:
+		return hack.Slice(json.DumpToString(value.GetMysqlJSON())), nil
```

Comment:
Could we imitate above style and implement the function `String()` for `json.JSON`?